.gitlab-ci/env.sh

```diff
@@ -2,46 +2,60 @@
 
 #
 
-export BRANCH_NAME=${BRANCH_NAME:=$CI_COMMIT_REF_NAME}
-export COMMIT=${COMMIT:=$CI_COMMIT_SHA}
+export BRANCH_NAME=${BRANCH_NAME:=$CI_COMMIT_REF_SLUG}
```

Comment:
what is the difference between `CI_COMMIT_REF_NAME` and `CI_COMMIT_REF_SLUG`

Reply:
`CI_COMMIT_REF_NAME` is the raw name 
`CI_COMMIT_REF_SLUG` is the slugify version (with all special char replaced by `-`

Reply:
As K8S doesn't like special char I use the slug here